frontend/src/Components/ProductTypes/Products.tsx

```diff
@@ -186,6 +192,39 @@ const Products = () => {
           transformData={transformIncomePaystubsData}
           />
       )}
+
+      {(products.includes("cra_base_report") || products.includes("cra_income_insights")) && (
+        <Endpoint
+          endpoint="/cra/check_report"
```

Comment:
This should be `/cra/check_report/base_report/get`

Reply:
Updating this endpoint to `/cra/get_base_report` instead. To avoid confusion, I want to use this naming convention since the frontend is not directly calling the plaid endpoint.